.github/workflows/deploy.yml

```diff
@@ -11,27 +11,30 @@ jobs:
     name: lint
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: Borales/actions-yarn@v2.3.0
+      - uses: actions/checkout@v3
+      - run: corepack enable
+      - run: yarn set version berry
```

Comment:
This is not necessary if using corepack... Did the workflows failed?

Reply:
could be moving up to node 16 fixed it, or checking in the .yarn/releases